themes/theme-gmd/pages/Favorites/components/FavoritesList/components/Item/components/ProductInfo/components/Characteristics/index.jsx

```diff
@@ -8,10 +8,11 @@ import PropTypes from 'prop-types';
  */
 const Characteristics = ({ characteristics }) => (
   <ul>
-    {characteristics.map(({ name, value }) => (
-      <li key={`${name}-${value}`}>
-        {name}: {value}
-      </li>
+    {characteristics.filter(({ name, value }) => !!name && !!value)
```

Comment:
It seems, that the shape of child product characteristics entries changed with the catalog extension. Before there was a property `name`, which doesn't exist anymore. So with catalog 2.0 there won't be any characteristics displayed anymore. But both extensions provide a property `label` which could be used to render characteristics.

![Bildschirmfoto 2019-07-11 um 09 45 26](https://user-images.githubusercontent.com/13087792/61032099-14884000-a3c1-11e9-8a7a-9592aae11e13.png)